docs/documentation/platform/secret-scanning.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Secret Scanning'
+description: "Scan and prevent secret leaks in your code repositories"
+---
+
+The Infisical Secret Scanner allows you to keep an overview and stay alert of exposed secrets across your entire GitHub organization and repositories.
+
+The Infisical Secret Scanner helps you detect exposed secrets that have already made it into your remote repositories. To further enhance security, we recommend you also use our [CLI Secret Scanner](/cli/scanning-overview#automatically-scan-changes-before-you-commit) to scan for exposed secrets prior to pushing your changes.
```

Comment:
Sounds a bit repetitive from the first paragraph, might be worth trying to rephrase the second paragraph